packages/wdio-snapshot-service/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@wdio/snapshot-service",
```

Comment:
I would love to have snapshot testing a core capability without having users adding an extra dependency. I think for DOM snapshot comparisons that should be possible. However if we go into image/screenshot snapshot comparisons where an image comparison dependency is needed, an extra service makes sense.

Reply:
Another point is as we need the test object to identity the running test file and name, which depending on the running framework like Mocha... So far I found the service beforeTest is the best place to put it

Reply:
You can implement a service as part of the `@wdio/runner` package and inject it into the runtime via `this._configParser.addService({ ... })`. It will give you access to all hooks like with services.

Reply:
thanks Christian lemme try this out

Reply:
well.. I think it make sense to keep it separate as the dependencies reason...